lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1529,6 +1538,32 @@ OpFoldResult Aten__Contains__StrOp::fold(ArrayRef<Attribute> operands) {
   return nullptr;
 }
 
+//===----------------------------------------------------------------------===//
+// Aten__Contains__IntListOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult Aten__Contains__IntListOp::fold(ArrayRef<Attribute> operands) {
+  auto arrayConstruct = isListConstructNotModified(l());
+  auto itemConstruct = item();
+  if (!arrayConstruct)
+    return nullptr;
+
+  int64_t item;
+  SmallVector<int64_t> list;
+
+  if (!matchPattern(itemConstruct, m_TorchConstantInt(&item)))
+    return nullptr;
```

Comment:
You can directly use `item()` here.

Reply:
I like this too, since it makes it symmetric with the `matchPattern` below

Reply:
@JakopinA  ^